gitpod-io/website
# docs: `gp docs` in Gitpod CLI

## Description

Related https://github.com/gitpod-io/gitpod/pull/15724

## Related Issue(s)



Fixes #

## How to test


- Open Preview
- Go to `/docs/references/gitpod-cli` & see :eyes:

## Release Notes



```release-note
NONE
```
